test/e2e/generate_systemd_test.go

```diff
@@ -71,4 +72,17 @@ var _ = Describe("Podman generate systemd", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	It("podman generate systemd pod", func() {
+		n := podmanTest.Podman([]string{"pod", "create", "--name", "foo"})
+		n.WaitWithDefaultTimeout()
+		Expect(n.ExitCode()).To(Equal(0))
+
+		n = podmanTest.Podman([]string{"create", "--pod", "foo", "alpine", "top"})
+		n.WaitWithDefaultTimeout()
+		Expect(n.ExitCode()).To(Equal(0))
+
+		session := podmanTest.Podman([]string{"generate", "systemd", "foo"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
y'know, this thing about "just check exit status but don't bother looking at output" troubles me greatly. Would it be possible to at least do something like look for 'Unit', 'ExecStart', 'WantedBy' in stdout?

Reply:
I agree. I will add some tests now!

Reply:
Added a commit to improve the tests. I'll add some unit tests on top as well.